app/graphql/types/query_type.rb

```diff
@@ -17,6 +17,7 @@ def therapists(labels: nil)
         Therapist.all.each do |therapist|
           labels.each do |label|
             if therapist.labels.include?(label)
+              binding.pry
```

Comment:
here's the pesky pry!!!!

Reply:
I believe so, I think that rails has a method to ensure that the data type comes in a way that we prefer

Reply:
@torlemann - pry has been deleted and will be removed with the next push